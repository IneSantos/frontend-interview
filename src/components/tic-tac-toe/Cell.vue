<template>
    <div class="cell" @click="$emit('click')">
      <span v-if="value !== ''">{{ value }}</span>
    </div>
</template>

<script>
  export default {
    name:'Cell',
    props: {
      value: String
    }
  }
</script>

<style lang="scss">
  .cell {
    width: 64px;
    height: 64px;
    border: 2px solid #D8D8D8;
    font-size: 42px;
    display: flex;
    align-items: center;
    justify-content: center;

     @media only screen and (min-width: 768px) {
        width: 132px;
        height: 132px;
     }

  }
</style>