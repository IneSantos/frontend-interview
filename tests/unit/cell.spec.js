import { shallowMount } from '@vue/test-utils'
import Cell from '@/components/tic-tac-toe/Cell.vue'

describe('HelloWorld.vue', () => {
  it('renders default', () => {
    const wrapper = shallowMount(Cell);
    expect(wrapper).toMatchSnapshot();
  })
})
