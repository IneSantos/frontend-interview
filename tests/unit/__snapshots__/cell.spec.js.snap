// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue renders default 1`] = `<div class="cell"><span></span></div>`;

exports[`HelloWorld.vue renders props.msg when passed 1`] = `<div class="cell"><span></span></div>`;
